### PR TITLE
Added instructions to run the server

### LINKED ISSUE #33: Add instructions to run the server
`README.md` should be enhanced to explain how to run the Neubot server.

### PR DESCRIPTION
**Pull request by DavideAllavena**
Closes #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,4 +25,42 @@ that is personal data under the European law.  For more details
...
+
+In order to start the server you need root privileges.
+
+Assuming you are in the root directory of neubot-server, you can simply start
+the server running:
+
+```BASH
+$ sudo bin/neubot-server-dev
+```
+All the loaded modules will be propted.
```
Comment (bassosimone) [inline README.md]: Do you mean `printed`?

Comment (DavideAllavena) [inline README.md]: yes, sorry

**Comment by DavideAllavena** [comment]
We need to talk about the list available backends.

File: README.md [hunk 2]
```diff
@@ -25,4 +25,42 @@ that is personal data under the European law.  For more details
...
+```
+
+Accepts the following options:
+
+* `-d ` Do not demonize neubot-server.
+
+* `-v` Makes the command more verbose.
+
+
+### Known issues
```
Comment (bassosimone) [inline README.md]: I'd not include this part in the README.md. Rather, I'd fix the warning. (See related issue #55).

**Comment by bassosimone** [comment]
Thank you! I have tweaked the diff a bit, to provide more information regarding why root privileges are needed (people is usually suspicious about running something as root). This can now be merged!

**Pull request merged**